### ISSUE TITLE
Buffer overflow?

### ISSUE THREAD

**Issue by Azhrei** [nriley/launch#1]
I got the following while building launch-1.1 from source (since that's apparently the latest that fink ships with):

```
/sw/src/fink.build/launch-1.1-1004/launch-1.1/main.c:655:2: warning: array index
      4 is past the end of the array (which contains 4 elements)
      [-Warray-bounds]
        tmpBuffer[4] = '\0';
        ^         ~
/sw/src/fink.build/launch-1.1-1004/launch-1.1/main.c:654:2: note: array
      'tmpBuffer' declared here
        static char tmpBuffer[4];
        ^
```

Hope that helps. :)

**Comment by nriley** [comment]
This seems like more of a Fink issue then...  `launch` 1.1 is 10 years old (!).  I did fix this bug in 2013 ( 6c382d7017804538d8d16e7c2e46e608a3cf5551).

**Comment by Azhrei** [comment]
Wow, definitely a fink problem then.  Thanks for the quick response!
